docs/developers/tutorials/staking-contract.md

```diff
@@ -353,35 +334,18 @@ This is NOT an error. This simply means you provided way more gas than needed, s
 
 ## The first stake
 
-Let's add a snippet for the staking function:
+Let's call the stake function:
 
 ```bash
-USER_PEM="~/Downloads/tutorialKey.pem"
-PROXY="https://devnet-gateway.multiversx.com"
-CHAIN_ID="D"
-
-SC_ADDRESS=erd1qqqqqqqqqqqqq...
-STAKE_AMOUNT=1
-
-deploy() {
-    mxpy --verbose contract deploy --project=${PROJECT} \
-    --recall-nonce --pem=${USER_PEM} \
+mxpy --verbose contract call ${SC_ADDRESS}... \
+    --proxy=https://devnet-gateway.multiversx.com --chain=D \
+    --send --recall-nonce --pem=/Downloads/tutorialKey.pem \
     --gas-limit=10000000 \
-    --send --outfile="deploy-devnet.interaction.json" \
-    --proxy=${PROXY} --chain=${CHAIN_ID} || return
-}
-
-stake() {
-    mxpy --verbose contract call ${SC_ADDRESS} \
-    --proxy=${PROXY} --chain=${CHAIN_ID} \
-    --send --recall-nonce --pem=${USER_PEM} \
-    --gas-limit=10000000 \
-    --value=${STAKE_AMOUNT} \
+    --value=1 \
```

Comment:
is the value nominated later? I don't think so. then you should use `1000000000000000000` for 1 EGLD